lib/query-interface.js

```diff
@@ -520,10 +520,11 @@ class QueryInterface {
 
     options.type = QueryTypes.UPSERT;
     options.raw = true;
+    options.instance = model;
```

Comment:
`instance` is named `model` in the parameter list, but I would like to keep the syntax of insert method.